build/compile/Microsoft.DotNet.Cli.Archive.targets

```diff
@@ -0,0 +1,49 @@
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
The duplicate name had me a little confused on this file. Perhaps `FirstTimeArchive.targets` ?